src/agent/communicator/include/http_client.hpp

```diff
@@ -35,17 +35,12 @@ namespace http_client
         boost::beast::http::response<boost::beast::http::dynamic_body>
         PerformHttpRequest(const HttpRequestParams& params) override;
 
-        std::optional<std::string> AuthenticateWithUuidAndKey(const std::string& host,
-                                                              const std::string& port,
-                                                              const std::string& uuid,
-                                                              const std::string& key,
-                                                              const bool useHttps) override;
+        std::optional<std::string>
+        AuthenticateWithUuidAndKey(const std::string& host, const std::string& uuid, const std::string& key) override;
```

Comment:
Can we change the `host` name on the first argument? Should be address or url now...

Reply:
Done

Reply:
This repeats in many other functions.

Reply:
I changed it in HttpRequestParams constructor as well. I may be missing other occurrences.